### ISSUE TITLE
Make the pypi page pretty

### ISSUE THREAD

**Issue by wasade** [biocore/biom-format#431]
...like the [scikit-bio](https://pypi.python.org/pypi/scikit-bio) one

**Comment by souravsingh** [comment]
Is the issue still open?If so,I would like to work on it.

**Comment by wasade** [comment]
No, pyqi is a dead end. Sorry for the confusion -- will close shortly

**Comment by wasade** [comment]
Oh, sorry! I misread, thought the subject was pyqi. Yes, this is still open and it would be great if you tackled it. Thanks!

**Comment by wasade** [comment]
This one shouldn't be too bad just a little more context. We do not need to put in a logo.
